testserver/minecraft.js

```diff
@@ -0,0 +1,72 @@
+// Require Node.js standard library function to spawn a child process
+const spawn = require('child_process').spawn;
+let minecraftServerProcess;
+
+var isMinecraftRunning = false;
+const MINECRAFT_STATUS = {
+  RUNNING: 'RUNNING',
+  NOT_RUNNING: 'NOT_RUNNING'
+};
+
+// Express.js app listening for HTTP requests
+
+var express = require('express');
+var app = express();
+var port = 3000;
+
+app.get('/', (req, res) => res.send('MinecraftExpressApp is up and running!'));
+
+app.get('/status', (req, res) => {
+  // talk to Minecraft to get status
+  const status = getMinecraftStatus();
+  res.send({ status: status });
+});
+
+app.post('/start', (req, res) => {
+  if (!isMinecraftRunning) {
+    startMinecraft();
+    isMinecraftRunning = true;
+  }
+  res.send({ start: true });
+});
+
+app.delete('/shutdown', (req, res) => {
+  if (isMinecraftRunning) {
+    shutdownMinecraft();
+    isMinecraftRunning = false;
```

Comment:
I'd move this into the `shutdownMinecraft` method right at the end.

Reply:
Done.